ddtrace/tracer/option.go

```diff
@@ -446,6 +453,9 @@ func newConfig(opts ...StartOption) *config {
 		}
 		c.dogstatsdAddr = addr
 	}
+	if c.debugOpenSpans {
+		c.spanTimeout = internal.DurationEnv("DD_TRACE_OPEN_SPAN_TIMEOUT", time.Second)
```

Comment:
So just to clarify, `DD_TRACE_OPEN_SPAN_TIMEOUT` is both the max age of a span before it gets logged as abandoned, and it's how often the log ticker goes off? I wonder if those should be separate, or the log ticker should perhaps not be configurable at all. What are your thoughts here?

Reply:
Yes, it was the same for both, but I agree, especially if the logger will have to log every second as it currently is. It might be closer to expected behavior if it wasn't impacted by the user's configuration, as well. I can change it to a constant like 30 seconds-1 minute or so.